lib-tui/GHCup/Brick/Common.hs

```diff
@@ -179,6 +180,13 @@ notInstalledSign = "X "
 notInstalledSign = "✗ "
 #endif
 
+checkBoxSelectedSign :: String
+#if IS_WINDOWS
```

Comment:
We already have a function for that: `isWindows`. We don't need the explicit CPP everywhere.

Reply:
Fixed